addons/website_profile/views/website_profile.xml

```diff
@@ -126,7 +126,7 @@
                             <div class="row">
                                 <input name="user_id" t-att-value="user.id" type="hidden"/>
                                 <div class="form-group col-12">
-                                    <label class="text-primary mb-1 d-block" for="user_name"><span class="font-weight-bold">Real name</span></label>
+                                    <label class="text-primary mb-1 d-block" for="user_name"><span class="font-weight-bold">Name</span></label>
```

Comment:
Even is it's a tiny change, you should add a small note about it in your commit message.